common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1439,6 +1439,22 @@ def test_join_team(self):
         self.teams_page.click_all_topics()
         self.verify_my_team_count(1)
 
+    def test_join_team_from_teams_page(self):
```

Comment:
It would be good to fix the tests so that they don't visit pages directly. It means that they aren't really testing the behavior as the user will see it. It makes sense to also do a refresh to verify that the change took, but that should be a secondary test IMO.